src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -26,4 +26,16 @@ public Movie get(Long id) {
     public List<Movie> getAll() {
         return movieDao.getAll();
     }
+
+    @Override
+    public Boolean update(Movie movie) {
+        Movie currentMovie = get(movie.getId());
+        movieDao.update(movie);
+        return currentMovie.equals(get(movie.getId()));
```

Comment:
The logic here is inefficient. Instead of checking the database again, consider returning `true` after a successful update operation.